pages/common/evil-winrm.md

```diff
@@ -0,0 +1,37 @@
+# evil-winrm
+
+> Windows Remote Management (WinRM) shell for pentesting.
+> Once connected, we get a PowerShell prompt on the target host.
+> More information: <https://github.com/Hackplayers/evil-winrm>.
+
+- Connect to a host:
+
+`evil-winrm --ip {{ip}} --user {{user}} --password {{password}}`
+
+- Connect to a host, passing the password hash:
+
+`evil-winrm --ip {{ip}} --user {{user}} --hash {{nt_hash}}`
+
+- Connect to a host, specifying folders for scripts and executables:
+
+`evil-winrm --ip {{ip}} --user {{user}} --password {{password}} --scripts {{path/to/scripts}} --executables {{path/to/executables}}`
+
+- Connect to a host, using SSL:
+
+`evil-winrm --ip {{ip}} --user {{user}} --password {{password}} --ssl --pub-key {{path/to/pubkey}} --priv-key {{path/to/privkey}}`
+
+- Upload a file to the host:
+
+`*Evil-WinRM* PS C:\Users\user\Documents> upload {{path/to/local/file}} {{path/to/remote/file}}`
```

Comment:
Is there any reason why this is different to the previous example? If this is a shell command for `evil-winrm`, perhaps we can represent that more concisely, like this?
```suggestion
`PS > upload {{path/to/local/file}} {{path/to/remote/file}}`
```

Same goes  for the other examples if I'm understanding this correctly.

Reply:
That's up to you. My idea is to show the prompt like the user will see when she gets a shell. I've put the Windows path as well as it is where the user lands when Evil-WinRM can connect to the target.
Also, putting `PS > ` confuses with a normal PowerShell prompt, no ? What do you think about this ?

Reply:
Ah, I see. The prompt is a bit long though. What about this?
```suggestion
`*Evil-WinRM* PS > upload {{path/to/local/file}} {{path/to/remote/file}}`
```